app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseBottomSheet.kt

```diff
@@ -67,7 +67,9 @@ abstract class BaseBottomSheet<K : ScreenKey, B : ViewBinding, M : Parcelable, E
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setStyle(DialogFragment.STYLE_NORMAL, org.simple.clinic.R.style.Clinic_V2_Theme_BottomSheetFragment)
+    // TODO : It's not necessary since we applied the default style in theme.
+    //  Once we can verify the bottom sheet with new navigation framework. We can remove this.
```

Comment:
One thing I have started doing is no longer adding `TODO` statements in the code and putting them in Clubhouse with a `maintenance` label since nobody looks at `TODO` statements. Can we do that here?